pkg/auth/dependency/newinteraction/nodes/create_identity_begin.go

```diff
@@ -35,35 +29,20 @@ func (n *NodeCreateIdentityBegin) Apply(perform func(eff newinteraction.Effect)
 func (n *NodeCreateIdentityBegin) DeriveEdges(ctx *newinteraction.Context, graph *newinteraction.Graph) ([]newinteraction.Edge, error) {
 	var edges []newinteraction.Edge
 	for _, t := range ctx.Config.Authentication.Identities {
-		if n.RequestedIdentity != nil && n.RequestedIdentity.Type != t {
-			continue
-		}
-
 		switch t {
 		case authn.IdentityTypeAnonymous:
 			panic("TODO(interaction): handle anonymous signup")
-
 		case authn.IdentityTypeLoginID:
-			for _, c := range ctx.Config.Identity.LoginID.Keys {
-				if n.RequestedIdentity != nil &&
-					n.RequestedIdentity.Claims[identity.IdentityClaimLoginIDKey] != c.Key {
-					continue
-				}
-				edges = append(edges, &EdgeCreateIdentityLoginID{Config: c, RequestedIdentity: n.RequestedIdentity})
-			}
+			edges = append(edges, &EdgeUseIdentityLoginID{
+				IsCreating: false,
```

Comment:
Just a note: adding identity in settings page would most likely need this node, we should fix it later (IsCreating: true).